jinja2/filters.py

```diff
@@ -445,7 +445,7 @@ def do_urlize(eval_ctx, value, trim_url_limit=None, nofollow=False,
     return rv
 
 
-def do_indent(s, width=4, indentfirst=False):
+def do_indent(s, width=4, indentfirst=False, indentblanklines=False):
```

Comment:
If you want to, also change the signature to `do_indent, s, width=4, indent_first=False, indent_blank_lines=False, indentfirst=None)` and raise a deprecation warning if `indentfirst` is not None. Not required though, I can follow up with this after merging.